src/content/developers/docs/scaling/state-channels/index.md

```diff
@@ -6,71 +6,256 @@ sidebar: true
 
-| Pros                                                                             | Cons                                                                                                                                                |
-| -------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------- |
-| Instant withdrawal/settling on Mainnet (if both parties to a channel cooperate). | Time and cost to set up and settle a channel - not so good for occasional one-off transactions between arbitrary users.                             |
-| Extremely high throughput is possible.                                           | Need to periodically watch the network (liveness requirement) or delegate this responsibility to someone else to ensure the security of your funds. |
-| Lowest cost per transaction - good for streaming micropayments.                  | Have to lockup funds in open payment channels.                                                                                                      |
-|                                                                                  | Don't support open participation.                                                                                                                   |
+To solve this problem, "virtual channels" were created. Unlike regular channels that require on-chain transactions to open and terminate, a virtual channel can be opened, executed, and finalized without interacting with the main chain. It is even possible to settle disputes off-chain using this method.  
+
+This system relies on the existence of so-called "ledger channels", which have been funded on-chain. Virtual channels between two parties can be built on top of an existing ledger channel, with the owner(s) of the ledger channel serving as an intermediary. 
+
+Users in each virtual channel interact via a new contract instance, with the ledger channel able to support multiple contract instances. The ledger channel's state also contains more than one contract storage state, allowing for parallel execution of applications off-chain between different users.  
+
+Just like regular channels, users exchange state updates to progress the state machine. Except a dispute arises, the intermediary only has to be contacted when opening or terminating the channel. 
+
+### Virtual payment channels {#virtual-payment-channels}
+
+Virtual payment channels work off the same idea as virtual state channels: participants connected to the same network can pass messages without needing to open a new channel on-chain. In virtual payment channels, value transfers are routed through one or more intermediaries, with guarantees that only the intended recipient can receive transferred funds. 
+
+## Applications of state channels {#applications-of-state-channels}
+
```

Comment:
Might be nice to mention the potential for implementing a state-channel on a layer-2.

Reply:
I thought scaling solutions that don't use Ethereum for data availability don't fall under layer-2?

Reply:
sure, but I remember discussions about whether a state-channel could be implemented posting its initial and final state to an L2 for additional scaling. 

e.g. Raiden on Arbitrum https://medium.com/raiden-network/the-raiden-network-now-live-on-arbitrum-3d57fedf7961

Reply:
A simple one-liner like that is plenty in my opinion.

Reply:
Here's the final version I added:

> Implementing state channels on layer 2 solutions, such as [rollups](/developers/docs/scaling/#rollups), could make them even more attractive for payments. While channels offer cheap payments, the costs of setting up the on-chain contract on Mainnet during the opening phase can be get expensive—especially when gas fees spike. Ethereum-based rollups offer [lower transaction fees](https://l2fees.info/) and can reduce overhead for channel participants by bringing down setup fees.

Reply:
nice